CRM/Mosaico/BAO/MosaicoTemplate.php

```diff
@@ -72,6 +72,9 @@ public static function findBaseTemplates($ignoreCache = FALSE, $dispatchHooks =
         }
       }
 
+      // get list of base templates that needs be to hidden from the UI
+      $templatesToHide = CRM_Core_BAO_Setting::getItem('Mosaico Preferences', 'mosaico_hide_base_templates');
```

Comment:
@kurund can we use \Civi::settings()->get('mosaico_hide_base_templates') here ?

Reply:
Thanks for reviewing. I have made the changes.